packages/js/product-editor/src/contexts/validation-context/README.md

```diff
@@ -0,0 +1,39 @@
+# Validations and error handling
+
+This directory contains some components and hooks used for validations in the product editor.
+
+See [Product Editor Development Handbook](../../../../../../docs/product-editor-development/README.md)
+for more information.
+
+## Add a link to error snackbar
+
+### What happens when there is an error in a field?
+
+1. Every field [will be validated](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/product-editor/src/contexts/validation-context/validation-provider.tsx#L87-L110).
```

Comment:
Is this true? I don't think this is strictly the case right?
To my understanding only the fields that make use of `useValidation` or `useValidations` hooks get registered and will get validated?

So maybe we can rephrase this:
`Fields registered in the validator context will get validated. A field can be registered by making use of the useValidation hook`
Maybe show a quick example as a sub-bullet.

Reply:
Done in [be38da9](https://github.com/woocommerce/woocommerce/pull/49723/commits/be38da928a6429fd51eed853c2fe3eebe15e30f9)